wiki/start/a-simple-game.md

```diff
@@ -5,605 +5,1091 @@ redirect_from:
+Before we can test our game, we should set the size of the desktop window. Any configuration that needs to happen for the desktop version of your game needs to be set in the LWJGL3Launcher class. Find this file in the project folder:
 
-## Configuring the Starter Classes
-Given our requirements we can now configure our different starter classes. We'll start with the **desktop project**. Open the `Lwjgl3Launcher.java` class in `lwjgl3/src/…` (or `drop-lwjgl3` under Eclipse). We want a 800x480 window and set the title to "Drop". The code should look like this:
+![Lwjgl3Launcher](/assets/images/dev/a-simple-game/2.png)
 
 ```java
-package com.badlogic.drop;
+...
 
-import com.badlogic.gdx.backends.lwjgl3.Lwjgl3Application;
-import com.badlogic.gdx.backends.lwjgl3.Lwjgl3ApplicationConfiguration;
-import com.badlogic.drop.Drop;
+private static Lwjgl3ApplicationConfiguration getDefaultConfiguration() {
+    Lwjgl3ApplicationConfiguration configuration = new Lwjgl3ApplicationConfiguration();
+    configuration.setTitle("Drop");
+    configuration.useVsync(true);
+    configuration.setForegroundFPS(Lwjgl3ApplicationConfiguration.getDisplayMode().refreshRate + 1);
+    configuration.setWindowedMode(800, 500); //this line changes the size of the window
+    configuration.setWindowIcon("libgdx128.png", "libgdx64.png", "libgdx32.png", "libgdx16.png");
 
-public class Lwjgl3Launcher {
-   public static void main (String[] arg) {
-      Lwjgl3ApplicationConfiguration config = new Lwjgl3ApplicationConfiguration();
-      config.setTitle("Drop");
-      config.setWindowedMode(800, 480);
-      config.useVsync(true);
-      config.setForegroundFPS(60);
-      new Lwjgl3Application(new Drop(), config);
-   }
+    return configuration;
 }
 ```
 
-If you are only interested in desktop development, you can skip the rest of this section.
-{: .notice--info}
+## Loading Assets
+2D games made in libGDX need assets: images, audio, and other resources that comprise the project. In this case, we'll need a bucket, a raindrop, a background, a water drop sound effect, and music. If you're pretty resourceful, you can make these on your own. For simplicity's sake, you can download these examples which are optimized for this tutorial (Right click > Save as)
+
+[bucket.png](/assets/downloads/tutorials/simple-game/bucket.png)<br>
+[drop.png](/assets/downloads/tutorials/simple-game/drop.png)<br>
+[background.png](/assets/downloads/tutorials/simple-game/background.png)<br>
+[drop.mp3](/assets/downloads/tutorials/simple-game/drop.mp3)<br>
+[music.mp3](/assets/downloads/tutorials/simple-game/music.mp3)
```

Comment:
I would have preferred zip too since it forces it to be a download instead of opening in the browser. However, I've dealt with too many newbies who don't know how to unzip a zip file. Young and old, aspiring programmers and non-programmers alike. I just don't want to deal with that in their first tutorial.

Reply:
Oh, if you don't want left-click to open in a browser, just add the `download` attribute. Here is an example of how you can do this:

https://github.com/libgdx/libgdx.github.io/blob/2a7d8ec/wiki/input/cursor-visibility-and-catching.md?plain=1#L17

`?nomagnify` prevents the wiki from doing an image gallery preview thing no-one wants. Specifying a filename is optional.